app/models/translation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Translation < ActiveRecord::Base
+  belongs_to :resource
+  belongs_to :language
+  has_many :custom_pages
+
+  def add_new_version
+    Translation.create(resource: resource, language: language, version: version + 1)
+  end
+
+  def self.latest_translation(resource_id, language_id)
```

Comment:
See http://guides.rubyonrails.org/active_record_querying.html#scopes
```ruby
class Translation < ActiveRecord::Base
    scope :latest_for, -> (resource_id, language_id) {
        where(resource_id: resource_id, language_id: language_id).order(version: :desc)
    }
end
```
This would create a `Translation.latest_for(resource_id, language_id)` method.
@soberstadt Would you recommend a scope here?